WKData/Sources/WKData/Data Controllers/WKGrowthTasks/WKGrowthTasksDataController.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+public final class WKGrowthTasksDataController {
+
+    var service = WKDataEnvironment.current.mediaWikiService
+    let project: WKProject
+
+    public init (project: WKProject) {
+        self.project = project
+    }
+
+    // MARK: GET Methods
+
+    // TODO: to get more tasks, send used IDS to `ggtexcludepageids` parameter
+    // TODO: remove debug prints
+    public func getGrowthAPITask(task: WKGrowthTaskType = .imageRecommendation, completion: @escaping (Result<[WKGrowthTask.Page], Error>) -> Void) {
+
+        guard let service else {
+            return
```

Comment:
It'd be good to call the error handler for any early returns. I've got some more generalized ones in `WKDataControllerError`, so you can use `completion(.failure(WKDataControllerError.mediaWikiServiceUnavailable)` here.